tests/linearizability/failpoints.go

```diff
@@ -37,25 +37,25 @@ const (
 
 var (
 	KillFailpoint                            Failpoint = killFailpoint{}
-	DefragBeforeCopyPanic                    Failpoint = goPanicFailpoint{"backend/defragBeforeCopy", triggerDefrag, AnyMember}
-	DefragBeforeRenamePanic                  Failpoint = goPanicFailpoint{"backend/defragBeforeRename", triggerDefrag, AnyMember}
-	BeforeCommitPanic                        Failpoint = goPanicFailpoint{"backend/beforeCommit", nil, AnyMember}
-	AfterCommitPanic                         Failpoint = goPanicFailpoint{"backend/afterCommit", nil, AnyMember}
-	RaftBeforeSavePanic                      Failpoint = goPanicFailpoint{"etcdserver/raftBeforeSave", nil, AnyMember}
-	RaftAfterSavePanic                       Failpoint = goPanicFailpoint{"etcdserver/raftAfterSave", nil, AnyMember}
-	BackendBeforePreCommitHookPanic          Failpoint = goPanicFailpoint{"backend/commitBeforePreCommitHook", nil, AnyMember}
-	BackendAfterPreCommitHookPanic           Failpoint = goPanicFailpoint{"backend/commitAfterPreCommitHook", nil, AnyMember}
-	BackendBeforeStartDBTxnPanic             Failpoint = goPanicFailpoint{"backend/beforeStartDBTxn", nil, AnyMember}
-	BackendAfterStartDBTxnPanic              Failpoint = goPanicFailpoint{"backend/afterStartDBTxn", nil, AnyMember}
-	BackendBeforeWritebackBufPanic           Failpoint = goPanicFailpoint{"backend/beforeWritebackBuf", nil, AnyMember}
-	BackendAfterWritebackBufPanic            Failpoint = goPanicFailpoint{"backend/afterWritebackBuf", nil, AnyMember}
-	CompactBeforeCommitScheduledCompactPanic Failpoint = goPanicFailpoint{"mvcc/compactBeforeCommitScheduledCompact", triggerCompact, AnyMember}
-	CompactAfterCommitScheduledCompactPanic  Failpoint = goPanicFailpoint{"mvcc/compactAfterCommitScheduledCompact", triggerCompact, AnyMember}
-	CompactBeforeSetFinishedCompactPanic     Failpoint = goPanicFailpoint{"mvcc/compactBeforeSetFinishedCompact", triggerCompact, AnyMember}
-	CompactAfterSetFinishedCompactPanic      Failpoint = goPanicFailpoint{"mvcc/compactAfterSetFinishedCompact", triggerCompact, AnyMember}
-	CompactBeforeCommitBatchPanic            Failpoint = goPanicFailpoint{"mvcc/compactBeforeCommitBatch", triggerCompact, AnyMember}
-	CompactAfterCommitBatchPanic             Failpoint = goPanicFailpoint{"mvcc/compactAfterCommitBatch", triggerCompact, AnyMember}
-	RaftBeforeLeaderSendPanic                Failpoint = goPanicFailpoint{"etcdserver/raftBeforeLeaderSend", nil, Leader}
+	DefragBeforeCopyPanic                    Failpoint = goPanicFailpoint{"defragBeforeCopy", triggerDefrag, AnyMember}
```

Comment:
per previous discussion, should the `defragBeforeCopy` changed to `backendDefragBeforeCopy`?

Of course, can be discussed in a separate session.